xpath-outcome-initiator/src/main/java/org/cristalise/kernel/persistency/outcomebuilder/field/StringField.java

```diff
@@ -300,7 +301,7 @@ public JSONObject generateNgDynamicFormsCls() {
         fieldElement.put("label", "ui-widget");
 
         JSONObject fieldGrid = new JSONObject();
-        fieldGrid.put("container", "ui-g");
+        fieldGrid.put("container", ngcontainer!=null? ngcontainer : "ui-g");

```

Comment:
use `StringUtils.isNotBlank(ngcontainer)`

Reply:
will apply this